pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -30,6 +30,11 @@ Rate limiting happens on a per-pull basis regardless of whether the pulled image
 
 For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
 
+## New features
```

Comment:
@russurquhart1 this section needs to be reformatted to look like this:

https://docs.d2iq.com/dkp/kommander/1.3/release-notes/1.3/#new-features